pkg/aws/policies.go

```diff
@@ -74,8 +74,13 @@ const (
 	ControlPlaneAccountRole = "instance_controlplane"
 	WorkerAccountRole       = "instance_worker"
 	SupportAccountRole      = "support"
-	OCMRole                 = "OCM"
-	OCMUserRole             = "User"
+
+	HCPInstallerRole = "installer"
```

Comment:
The keys are the same since both Hive roles and Hypershift roles are going to use the same trust policies.
`filename := fmt.Sprintf("sts_%s_trust_policy", file)`

Reply:
Was going to ask about that, won't they be different at some point? Will there be a refactor later or would you rather wait for the correct policies?

Reply:
Oh wait, this is for the trusted. This is true